utils/node.py

```diff
@@ -98,7 +98,7 @@ def find_nodes(node_tree, bl_idname):
 
 
 def update_opengl_materials(_, context):
-    if not hasattr(context, "object") or not context.object or not context.object.active_material:
+    if not hasattr(context, "object") or not context.object or not context.object.active_material or not context.material.luxcore.auto_mat_selection:
```

Comment:
Any reason why you use `context.material` here instead of the `context.object.active_material` that is used in the rest of this function?
They are probably the same in almost all cases, but it's a bit inconsistent.